index.rst

```diff
@@ -13,17 +13,312 @@
+  For the most part this is not a concern, since DP0.2 has been exercising those capabilities, but there is at least one relevant special case:
+
+  - It is planned to provide for time-series queries on moving objects in the RSP that appear together with appropriate cutouts
+    in the images on which the points in the time series were observed, i.e., following the object across the sky.
+    This would not be possible in DP0.3.
+  - More generally, the usual linkages from single-epoch measurements in catalog tables (in this case, ``DIASource`` and ``SSSource``)
+    to the associated images would not be available.
+
+- As the result of a static simulation, rather than a progressive nightly release through the PPDB as we will have in Operations,
+  DP0.3 would not exercise certain dynamic aspects of the data model, such as the transformation from DIAObject to SSObject when
+  sufficient data are accumulated to allow identification of a previous DIAObject as arising from an SSO.
+  It would therefore also not exercise the ability of users of the RSP to follow and/or replay that evolution.
+
+- In the real system, it will be possible to provide links from the Rubin-hosted tables directly to corresponding
+  external queries against the MPC.
+  Since the DP0.3 data include simulated SSOs, this will not be possible to exercise against the real MPC.
+
+.. _PREOPS-1152: https://jira.lsstcorp.org/browse/PREOPS-1152
+.. _RTN-011: https://rtn-011.lsst.io/
+
+Input Data
+==========
+
+The simulation itself is described at https://github.com/lsst-sssc/lsst-simulation .
+It produces the following tables, all of which have reference definitions in the `DPDD`_:
+
+.. _table-ssotables:
+
+.. table:: Major tables to be included in a Solar System DP0.3.  
+
+   +-----------+--------------------------------------------------+--------------+-------------------+---------------------+
+   | Table     | Purpose                                          | Size in rows | Number of columns | Data volume on disk |
+   |           |                                                  |              | (actual/DPDD)     | (in XXX format)     |
+   +===========+==================================================+==============+===================+=====================+
+   | DIASource | Observation in a single difference image         | ~1B          |                   |                     |
+   +-----------+--------------------------------------------------+--------------+-------------------+---------------------+
+   | SSSource  | Apparition of a reconstructed SSO in an image    | ~1B          |                   |                     |
+   +-----------+--------------------------------------------------+--------------+-------------------+---------------------+
+   | SSObject  | Reconstructed SSO with orbital parameters        | ~10M         |                   |                     |
```

Comment:
This might be an HEP-ism in my vocabulary.  Just to explain myself: I tend to use "detect" for something more "done in one place" - like "detected source on an image", and "reconstruct" for "concluded from combining several pieces of evidence".  E.g., "the transient was first detected on February 1st; by April it had been reconstructed as an SNIa candidate and its redshift measured".

But I'm happy to make the change you suggested.

Reply:
Ah right, if I heard "reconstructed particle track" I would probably have had the right association, but I think it's an unfamiliar phrasing in astronomy.